src/app/tutorial/tutorial.component.ts

```diff
@@ -12,23 +9,84 @@ export class TutorialComponent implements OnInit {
   currentStep: any | null = null; //if null, the tutorial will not show
   steps: any[] = [];
 
-  //********************************************
-  constructor(
-    private router: Router,
-    private route: ActivatedRoute,
-  ) {}
+  videos = [
+    {
+      id: '1_8elni4dv',
+      title: 'Workspace',
+      description:
+        'This is the workspace, a sandbox where you can design the molecules you want!',
+    },
+    {
+      id: '1_gchbjezj',
+      title: 'Block Tray',
+      description:
+        'This is the block tray, an area that stores Lego-like molecule blocks. You will use them to build your molecules.',
+    },
+    {
+      id: '1_g2u2woll',
+      title: 'Structure-Function',
+      description:
+        'Can can change the block information from structure to function view at any time. Click on an individual block to switch between function and structure modes',
```

Comment:
typo here: "Can can"

Reply:
the other descriptions end with a punctuation mark, so we should probably have one here, too, for consistency